홍순혁_2200015/소득불균형.py

```diff
@@ -0,0 +1,15 @@
+a = int(input())
+
+for i in range(1,a+1):
+    q=0
+    w= 0
+    b= int(input())
+    c = list(map(int,input().split()))
+    q = sum(c)//b
```

Comment:
```suggestion
    q = sum(c)/b
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄